src/main/java/uk/gov/ch/exception/TrustDataCountNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.ch.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
+public class TrustDataCountNotFoundException extends Exception{
```

Comment:
Yeah exactly just to avoid the duplication of code we decided to have the one exception class for all types of trust data- the error messages themselves will be more specific